src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -218,12 +218,11 @@ class HoldRequest extends React.Component {
         <div className="nypl-request-page-header">
           <div className="nypl-full-width-wrapper">
             <div className="row">
-              <div className="nypl-column-three-quarters">
+              <div className="nypl-column-full">
                 <Breadcrumbs
-                  query={searchKeywords}
+                  query={`q=${searchKeywords}`}
```

Comment:
I probably will just pass `searchKeywords` here. Leave the logic to add `q=` in <Breadcrumbs /> components. 

This way, the component will check `searchKeywords` once, so we don't need to check `searchKeywords` here to see if it is valid, and then in <Breadcrumbs /> we need to check if `q=` again.  And also, the component contains all the logic it needs itself.

But it is a rather minor suggestion.